### DISCUSSION TITLE
是的，我更换了昵称！ 🐱 ネコのメモ帳

### DISCUSSION THREAD

**Discussion by giscus** [MuelNova/blog#13] [Announcements]
# 是的，我更换了昵称！ 🐱 ネコのメモ帳

思考了许久这篇 blog 应该放在 小丑 专栏还是 blog 专栏，还是最后决定把一些生活中的小事（或者说不是那么有趣的）分享在这里。

https://n.ova.moe/blog/%e6%98%af%e7%9a%84-%e6%88%91%e6%9b%b4%e6%8d%a2%e4%ba%86%e6%98%b5%e7%a7%b0/

**Comment by MuelNova** [comment]
直到今日，我的昵称除了几个变种，目前在使用 `Mu Elnova` 或者 `μw nova`。
